layers/sqlsrv/test.php

```diff
@@ -1,7 +1,5 @@
 <?php
 
-putenv('ODBCSYSINI=/opt/microsoft/conf/');
```

Comment:
As mentioned in https://github.com/brefphp/extra-php-extensions/pull/474#issuecomment-1690764705, `putenv` doesn’t work.